### PR TITLE
doc: add doc for summary download as csv

### PR DESCRIPTION
**Pull request by guihardbastien**
Motivation:
Inform users on how they can download their summaries in a CSV format

Modification:
Update the report section with a few descriptive lines and some screenshots

### CODE REVIEW

File: content/reference/user/reports/index.md [hunk 1]
```diff
@@ -107,6 +107,17 @@ In flat mode, you can click on the title of each column to arrange the data in a
 
 {{< img src="summary.png" alt="Summary" >}}
 
+In order to download summaries as CSV for later use, you can click the download button on the top right side of the summary
```
Comment (Isammoc) [inline content/reference/user/reports/index.md:110]: Lacks a title to be able to directly link to this section.

 ```suggestion
### Download CSV

In order to download summaries as CSV for later use, you can click the download button on the top right side of the summary
```

File: content/reference/user/reports/index.md [hunk 2]
```diff
@@ -107,6 +107,17 @@ In flat mode, you can click on the title of each column to arrange the data in a
...
+
+{{< img src="download-summary.png" alt="Download-summary" >}}
+
+CSV files are generated according to the selected `Scenario`
```
Comment (Isammoc) [inline content/reference/user/reports/index.md:114]:
```suggestion
CSV files are generated according to the selected **Scenario**.
```

File: content/reference/user/reports/index.md [hunk 3]
```diff
@@ -107,6 +107,17 @@ In flat mode, you can click on the title of each column to arrange the data in a
...
+
+{{< img src="download-summary-select-scenario.png" alt="Download-summary-select-scenario" >}}
+
+When downloading a group level summary, you will receive data for both `Duration` and `Cumulated response time`.
```
Comment (Isammoc) [inline content/reference/user/reports/index.md:118]:
```suggestion
When downloading a group level summary, you will receive data for both **Duration** and **Cumulated response time**.
```

**Review by Isammoc — COMMENTED** [review]
backticks is for code
double star is for bold (and UI parts)

File: content/reference/user/reports/index.md [hunk 4]
```diff
@@ -107,6 +107,19 @@ In flat mode, you can click on the title of each column to arrange the data in a
...
 
+### Download CSV
+
+In order to download summaries as CSV for later use, you can click the download button on the top right side of the summary
+
+{{< img src="download-summary.png" alt="Download-summary" >}}
+
+CSV files are generated according to the selected **Scenario**
+
+{{< img src="download-summary-select-scenario.png" alt="Download-summary-select-scenario" >}}
```
Comment (jeremy-flusin) [inline content/reference/user/reports/index.md:118]: It feels weird that the Download CSV button is not visible on this screenshot.

**Review by Isammoc — APPROVED** [review]

**Review by jeremy-flusin — APPROVED** [review]

**Pull request merged**